Healthy/Resources/Info.plist

```diff
@@ -2,6 +2,19 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>ClientID</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.googleusercontent.apps.500241227951-jfe9f5o8li3l753c2146hqfru8aaa7o5</string>
```

Comment:
This is not the current app id

Reply:
I didn't get it 🤔.

Reply:
This is not the application client ID, we haven't create an app on Firebase yet!